src/Shared/Recruit.Vacancies.Client/Infrastructure/Repositories/MongoDbApplicationReviewRepository.cs

```diff
@@ -64,17 +64,29 @@ public Task UpdateAsync(ApplicationReview applicationReview)
                 new Context(nameof(UpdateAsync)));
         }
 
-        public async Task<List<T>> GetForEmployerAsync<T>(string employerAccountId)
+        public Task<List<ApplicationReviewCount>> GetStatusCountsForEmployerAsync(string employerAccountId)
         {
-            var filter = Builders<T>.Filter.Eq(EmployerAccountId, employerAccountId);
-            var collection = GetCollection<T>();
-
-            var options = new FindOptions<T> {Projection = GetProjection<T>() };
-
-            var result = await RetryPolicy.ExecuteAsync(context => collection.FindAsync<T>(filter, options),
-                new Context(nameof(GetForEmployerAsync)));
+            var collection = GetCollection<ApplicationReview>();
 
-            return await result.ToListAsync();
+            var aggregate = collection.Aggregate()
+
+                .Match(r => r.EmployerAccountId == employerAccountId
+                            && r.IsWithdrawn != true)

```

Comment:
`== false`

Reply:
This translates into `"isWithdrawn" : { "$ne" : true }` so handles if isWithdrawn is null.  I'll change it to use a non-lambda filter to save further confusion.